test/e2e_node/density_test.go

```diff
@@ -447,7 +447,8 @@ func runDensitySeqTest(f *framework.Framework, rc *ResourceCollector, testArg de
 // between creations there is an interval for throughput control
 func createBatchPodWithRateControl(f *framework.Framework, pods []*v1.Pod, interval time.Duration) map[string]metav1.Time {
 	createTimes := make(map[string]metav1.Time)
-	for _, pod := range pods {
+	for i := range pods {
+		pod := pods[i]
```

Comment:
Why are we using `pod := pods[i]` now instead of what range function returns us i.e., `_, pod := range pods`?

Reply:
see https://github.com/kubernetes/kubernetes/pull/106371#discussion_r748597098 `pod := pods[i]` is same as `pod := pod` when using `_, pod := range pods`

Reply:
@jonyhy96 I know they both are same, now I get it it's because of consistency in code, thanks for mentioning that comment.